index.html

```diff
@@ -0,0 +1,121 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>My Portfolio</title>
+    <link rel="stylesheet" href="./style.css" />
+    <link
+      rel="stylesheet"
+      href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.1.0/css/all.min.css"
+      integrity="sha512-10/jx2EXwxxWqCLX/hHth/vu2KY3jCF70dCQB8TSgNjbCVAC/8vai53GfMDrO2Emgwccf2pJqxct9ehpzG+MTw=="
+      crossorigin="anonymous"
+      referrerpolicy="no-referrer"
+    />
+    <link rel="icon" href="./assets/coding.png" type="image/x-icon" />
+  </head>
+  <body>
+    <header>
+      <div class="nav-desktop"></div>
+      <div class="nav-mobile">
+        <h2>Welcome</h2>
+        <div class="btn"><i class="fa-solid fa-bars"></i></div>
+      </div>
+    </header>
+
+    <section class="welcome-section">
+      <div class="welcome-header-text">
+        <h2>Hey there. I'm Kano</h2>
```

Comment:
- [ ] This is the main heading of your page so it would be best to place it in an `h1` tag for best semantic practices 💯 
- [ ] Also, it would be best to use the correct font and color as shown in the Figma template 👍 
- Your heading:
![Annotation 2022-03-17 183736](https://user-images.githubusercontent.com/85113567/158861794-f852332c-a45d-454f-8fa1-36d16f7d3a99.jpg)
- Figma design:
![Annotation 2022-03-17 183753](https://user-images.githubusercontent.com/85113567/158861838-bec9a7b3-4bd6-4021-b2fc-8856bf5f2abc.jpg)